projects/plugins/backup/src/js/components/Backups.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useEffect } from 'react';
+import { getDate, date, dateI18n } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import StatBlock from './StatBlock';
+import './backups-style.scss';
+import PostsIcon from './posts.svg';
+import CloudIcon from './cloud.svg';
+import UploadsIcon from './uploads.svg';
+import PluginsIcon from './plugins.svg';
+import ThemesIcon from './themes.svg';
+
+const Backups = () => {
+	// State information
+	const [ progress, setProgress ] = useState( null );
+	const [ trackProgress, setTrackProgress ] = useState( 0 );
+	const [ latestTime, setLatestTime ] = useState( '' );
+	const [ stats, setStats ] = useState( {
+		posts: 0,
+		uploads: 0,
+		plugins: 0,
+		themes: 0,
+	} );
+	const [ domain, setDomain ] = useState( '' );
+
+	const BACKUP_STATE = {
+		LOADING: 0,
+		IN_PROGRESS: 1,
+		NO_BACKUPS: 2,
+		COMPLETE: 3,
+	};
+
+	const [ backupState, setBackupState ] = useState( BACKUP_STATE.LOADING );
+
+	const progressInterval = 1 * 1000; // How often to poll for backup progress updates.
+
+	// One time loading
+	useEffect( () => {
+		setDomain( window.location.hostname );
```

Comment:
This may prove problematic on sites in a subdirectory. In the Jetpack plugin, we handled this by building the domain manually, like so:

https://github.com/Automattic/jetpack/blob/4f8ec4c3aacf7023bb5aa13056769dad09754736/projects/plugins/jetpack/_inc/client/state/initial-state/reducer.js#L62-L64
https://github.com/Automattic/jetpack/blob/829ee26a16e5ec36dc7a5310caabb7492d2a6230/projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php#L110
https://github.com/Automattic/jetpack/blob/529d07f441bb4667acd773918563296878e19cb4/projects/packages/status/src/class-status.php#L257-L282

I believe you should be able to require the Status package in the Backup plugin as well, to be able to use the `get_site_suffix` method.

Reply:
Thanks for the links there, updated to utilize the jetpack-status package.